Makefile

```diff
@@ -25,10 +25,10 @@ MAN_INSTALL_PATH := ${PREFIX}/share/man/man8/
 VERSION := ${shell cat ./VERSION}
 
 all: $(RUNC_LINK)
-	go build -i -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION}" -tags "$(BUILDTAGS)" -o runc .
+	go build -i -ldflags "-X version.gitCommit=${COMMIT} -X version.version=${VERSION}" -tags "$(BUILDTAGS)" -o runc .
```

Comment:
Revert this and the bottom change.

the fist part refers to the package, not the filename.